polars/polars-arrow/src/kernels/rolling/no_nulls/test.py

```diff
@@ -0,0 +1,274 @@
+    return k
+
+
+class Kurtosis(RollingObject):
+    """Iterator object that computes the kurtosis
+    of a rolling window over a Python iterable.
+    Parameters
+    ----------
+    iterable : any iterable object
+    window_size : integer, the size of the rolling
+        window moving over the iterable (must be
+        greater than 3)
+    Complexity
+    ----------
+    Update time:  O(1)
+    Memory usage: O(k)
+    where k is the size of the rolling window
+    Notes
+    -----
+    This implementation of rolling skewness is based
+    on the pandas code here:
+    https://github.com/pandas-dev/pandas/blob/master/pandas/_libs/window.pyx
+    """
+
+    def _init_fixed(self, iterable, window_size, **kwargs):
+        if window_size <= 3:
+            raise ValueError("window_size must be greater than 3")
+
+        self._buffer = deque(maxlen=window_size)
+        self._x1 = 0.0
+        self._x2 = 0.0
+        self._x3 = 0.0
+        self._x4 = 0.0
+
+        for new in islice(self._iterator, window_size - 1):
+            self._add_new(new)
+
+        # insert zero at the start of the buffer so that the
+        # the first call to update returns the correct value
+        self._buffer.appendleft(0)
+
+    def _init_variable(self, iterable, window_size, **kwargs):
+        if window_size <= 3:
+            raise ValueError("window_size must be greater than 3")
+
+        self._buffer = deque(maxlen=window_size)
+        self._x1 = 0.0
+        self._x2 = 0.0
+        self._x3 = 0.0
+        self._x4 = 0.0
+
+    def _add_new(self, new):
+        self._buffer.append(new)
+
+        self._x1 += new
+        self._x2 += new * new
+        self._x3 += new**3
+        self._x4 += new**4
+
+    def _remove_old(self):
+        old = self._buffer.popleft()
+
+        self._x1 -= old
+        self._x2 -= old * old
+        self._x3 -= old**3
+        self._x4 -= old**4
+
+    def _update_window(self, new):
+        old = self._buffer[0]
+        self._buffer.append(new)
+
+        self._x1 += new - old
+        self._x2 += new * new - old * old
+        self._x3 += new**3 - old**3
+        self._x4 += new**4 - old**4
+
+    @property
+    def current_value(self):
+        N = self._obs
+
+        if N <= 3:
+            return float("nan")
+
+        # compute moments
+        A = self._x1 / N
+        R = A * A
+
+        B = self._x2 / N - R
+        R *= A
+
+        C = self._x3 / N - R - 3 * A * B
+
+        R *= A
+
+        D = self._x4 / N - R - 6 * B * A * A - 4 * C * A
+
+        if B <= 1e-14:
+            return float("nan")
+
+        K = (N * N - 1) * D / (B * B) - 3 * ((N - 1) ** 2)
+
+        print(K)
+        # print(N, self._x3, A, B, C, D, K)
+
+        K = K / ((N - 2) * (N - 3))
+
+        print(K, ((N - 2) * (N - 3)))
+
+        return K
+
+    @property
+    def _obs(self):
+        return len(self._buffer)
+
+
+k = Kurtosis([3.0, 1.0, 4.0, 1.0, 5.0, 9.0, 2.0], 4)
+print(list(k))
```

Comment:
What is the purpose of this file?

If it is meant to test the code, I would suggest putting it in `py-polars/tests/` (with the other Python test code). Tests should also use `assert` instead of `print`.

If it is just a script you've been using as a sanity check, we probably don't need to commit it.